src/client/components/RoomSelection/index.js

```diff
@@ -0,0 +1,201 @@
+                roomName: roomName
+            }
+        });
+    }
+
+    return (
+        <Grid container style={GridContainerStyle}>
+
+            <Grid item xs={7}>
+                <div style={CenteredContainer}>
+                    <Button
+                        variant="contained"
+                        style={CreateRoomButtonStyle}
+                        onClick={handleClickOpen}
+                    >
+                        Create new room
+                    </Button>
+                </div>
+            </Grid>
+
+            {/* Table header */}
+            <Grid item xs={10}>
+                <div style={{ marginTop: "3em", marginLeft: "10px" }}>
+                    <span>Available rooms</span>
+                </div>
+            </Grid>
+            <Grid item xs={10} style={TableHeaderStyle}>
+                <Grid container style={{ height: "100%" }}>
+                    <Grid item xs={10}>
+                        <Paper style={PaperHeaderRowStyle}>
+                            <Grid container>
+                                <Grid item xs={5} style={TableColStyle}>
+                                    Room Name
+                                </Grid>
+                                <Grid item xs={5} style={TableColStyle}>
+                                    Room Host
+                                </Grid>
+                                <Grid item xs={2} style={TableColStyle}>
+                                    Players
+                                </Grid>
+                            </Grid>
+                        </Paper>
+                    </Grid>
+                </Grid>
+            </Grid>
+
+            {/* Table rows */}
+            {rows.map((row, i) => (
+                <Grid item xs={10} style={TableRowStyle} key={i}>
+                    <Grid container>
+                        <Grid item xs={10}>
+                            <Paper style={PaperRowStyle} elevation={2}>
+                                <Grid container>
+                                    <Grid item xs={5} style={TableColStyle}>
+                                        {row.roomName}
+                                    </Grid>
+                                    <Grid item xs={5} style={TableColStyle}>
+                                        {row.owner}
+                                    </Grid>
+                                    <Grid item xs={2} style={TableColStyle}>
+                                        {row.players}
+                                    </Grid>
+                                </Grid>
+                            </Paper>
+                        </Grid>
+                        <Grid item xs={2} style={TableColStyle}>
+                            <Button
+                                variant="contained"
+                                style={JoinRoomBtnStyle}
+                                onClick={() => { emitJoinGame(row.roomName, row.owner); JoinRoom(row.roomName) }}
```

Comment:
emitJoinGame take your name player as parameter instead of the roomOwnerName

Reply:
Oh okay i'll fix it